app.sample.json

```diff
@@ -0,0 +1,13 @@
+{
+    "expo": {
+        "extra": {
+            "IDPS": ["read.test"],
+            "LANGUAGE_CODE": "?"
```

Comment:
[App.js](https://github.com/biblemesh/toad-reader-apps/blob/db86f80be60fc6248d6eaf69b8714aafaf452314/App.js#L42) suggests `en` so perhaps this should include this as a suggestion